spec/models/proxy_order_spec.rb

```diff
@@ -196,6 +196,6 @@ def expect_cancelled_now(subject)
     # We still need to use be_within, because the Database timestamp is not as
     # accurate as the Rails timestamp. If we use `eq`, we have differing nano
     # seconds.
-    expect(subject.reload.canceled_at).to be_within(1.second).of Time.zone.now
+    expect(subject.reload.canceled_at).to be_within(2.second).of Time.zone.now
```

Comment:
the time dependency should be solved with [Timecop](https://github.com/travisjeffery/timecop). Did this fail in the build @luisramos0 ?

Reply:
with timecop.freeze, isn't it?
 I dont want to get into that now. what shall I do? revert this commit and create an issue?

Reply:
Sorry, I wasn't clear. Yep, let's create a tech debt issue, please.

Reply:
done here #3369